### ISSUE TITLE
SumatraPDF window is hidden when set viewer command manually.

### ISSUE THREAD

**Issue by lrtfm** [SublimeText/LaTeXTools#986]
On windows 10, Sublime 3.
When I set the viewer command as below for the inverse-search, I cannot see the SumatraPDF window:
```
{
    "viewer": "command",

    "viewer_settings": {
        // Platform-specific settings:

        "windows" : {
            // See README or third-party documentation
            "forward_sync_command":"SumatraPDF.exe -forward-search $src_file $line -inverse-search \"\\\"$sublime_binary\\\" \\\"%f:%l\\\"\" $pdf_file",
            "view_command":"SumatraPDF.exe -forward-search $src_file $line -inverse-search \"\\\"$sublime_binary\\\" \\\"%f:%l\\\"\" $pdf_file",
        },

    },
}
```
In file `LaTeXTools/viewers/command_viewer.py` line 129-131,
```
        external_command(
            command,
            cwd=os.path.split(pdf_file)[0],
        )
```
After I changed it to
```
        external_command(
            command,
            cwd=os.path.split(pdf_file)[0],
            show_window=True
        )
```
The window can show on the screen. 
Does we need add the line `show_window=True` to the file `LaTeXTools/viewers/command_viewer.py`?

**Comment by ig0774** [comment]
Thanks; this is a very good point. I think I'd like to try something a little bit more sophisticated, since we **do** want to show the window when using Sumatra and other viewers on Windows, but we don't when using `cmd`, e.g., to run `cmd /c open $pdf_file` or the like. Nevertheless, this is very fixable.

**Comment by ig0774** [comment]
This should be fixed in v3.12.12. Thanks for raising the issue.
